settings/com.endlessm.settings.gschema.override.in

```diff
@@ -24,6 +24,10 @@ icon-theme='EndlessOS'
 font-name='Lato 12'
 document-font-name='Lato Pro 12'
 
+# Declare the IM module as xim (X Input Module)
+[org.gnome.desktop.interface]
+gtk-im-module=xim
```

Comment:
We could not connect to ibus for some reason and found out that xim works in this case, and allows ibus IM (e.g. Pinyin) to work too...
Our intention is to investigate this further but for now we really need a solution that allows our users to use different IMs.